src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -117,6 +125,24 @@ public void setPerson(Person target, Person editedPerson) {
         addressBook.setPerson(target, editedPerson);
     }
 
+    @Override
+    public Person getPersonUsingName(Name nameOfPersonToGet,
+                                     boolean isFiltered) {
+        requireAllNonNull(nameOfPersonToGet, isFiltered);
+        Person personToGet = new Person(nameOfPersonToGet);
+        if (isFiltered && hasPersonInFilteredList(personToGet)) {
```

Comment:
```suggestion
        boolean hasPersonInFilteredList = hasPersonInFilteredList(personToGet);
        if (isFiltered && hasPersonInFilteredList) {
```
Is the usage of `isFiltered` with `hasPersonInFilteredList(personToGet)` in a single expression a violation of [SLAP](https://nus-cs2103-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard)?

_oops and I realised I did the same in_ `getModuleUsingModuleCode` _hor_